script/utils/sim-sequence.sh

```diff
@@ -126,10 +158,19 @@ NonceDisplayModified(){
     log_error "Nonce values are not available for one or more safes please investigate." 
     exit 99
   fi
+
+  if [[ $IS_3_OF_3 -eq 1 ]]; then
+    IS_3_OF_3=0 #Reset the 3_OF_3 flag
+    Chain_Governor_Safe="" # Reset the Chain_Governor_Safe
+  fi
```

Comment:
why do we need this block?

Reply:
This is useful to reset the IS_3_OF_3 at the end of the 3 of 3 nested tasks.
Otherwise, we will try to execute each time for nested (2/2) a 3/3 task.